start_esp/nginx-auto.conf.template

```diff
@@ -113,6 +113,24 @@ http {
 % endif
 
 % if location.proto == 'grpc':
+  % if cors_preset == 'basic':
+      # Begin basic CORS preset for GRPC backend
+      if (${'$'}request_method = 'OPTIONS') {
+        add_header 'Access-Control-Max-Age' 1728000;
```

Comment:
Consider making 'Access-Control-Max-Age' a customizable parameter, similar to other input parameters such as cors_allow_origin, cors_allow_methods, and cors_allow_headers.